vllm_spyre/platform.py

```diff
@@ -209,28 +198,16 @@ def check_and_update_config(cls, vllm_config: VllmConfig) -> None:
         os.environ["VLLM_DT_MAX_BATCH_SIZE"] = str(
             max(vllm_config.scheduler_config.max_num_seqs, 2))
 
-        # max product of batch size x tkv supported by the Spyre compiler
-        if ('granite-3.3-8b-instruct' in model_config.model
-                and parallel_config.world_size == 4):
-            # hard coded value for tensor parallel size 4 with the below model
-            # https://huggingface.co/ibm-granite/granite-3.3-8b-instruct
-            os.environ["VLLM_DT_MAX_BATCH_TKV_LIMIT"] = str(128 * 1024)
-            logger.info("Model granite-3.3-8b-instruct and tensor parallel " \
-            "size 4 detected. Using VLLM_DT_MAX_BATCH_TKV_LIMIT = %d",
-            128 * 1024)
-
-            # If no HDMA p2psize override was specified, set 256MB
-            if not os.getenv("FLEX_HDMA_P2PSIZE", None):
-                os.environ["FLEX_HDMA_P2PSIZE"] = str(1024 * 1024 * 256)
-                logger.info(
-                    "Model granite-3.3-8b-instruct and tensor parallel size 4 "
-                    "detected. Using FLEX_HDMA_P2PSIZE = %d",
-                    1024 * 1024 * 256)
-        else:
-            # default value for any other model/ tensor parallel size
+        # Hardcode some things for granite-3.3-8b-instruct
+        if cls.is_granite_3_8b(vllm_config.model_config):
+            cls.configure_granite_3_8b(vllm_config)
+
+        if not os.getenv("VLLM_DT_MAX_BATCH_TKV_LIMIT", None):
```

Comment:
NIT:
```suggestion
        if not os.getenv("VLLM_DT_MAX_BATCH_TKV_LIMIT"):
```
The default `default` for `os.getenv` is `None`. Maybe it is just me but adding the explicit default makes it harder for me to read.